llama-index-core/pyproject.toml

```diff
@@ -69,6 +69,7 @@ tqdm = "^4.66.1"
 pillow = ">=9.0.0"
 PyYAML = ">=6.0.1"
 wrapt = "*"
+pydantic = "^2.8.2"
```

Comment:
we might want to make sure this is the lowest we can go (if we can go lower, it just plays a little nicer with other packages)

Ideally we have `>=2.0.0,<3.0.0`

Reply:
sounds good to me, I can make that change.